### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by alastairong**
- Fixes flakey tests caused by lair needing more time to get ready and api-nginx dependencies. 
- Bumps tests for new holochain
- Bump holo-envoy and enable it for testing
- Add a 2.5 second sleep to envoy start up

I have run this many times without failure :)

### CODE REVIEW

File: modules/services/hpos-admin-api.nix [hunk 1]
```diff
@@ -18,8 +18,8 @@ in
 
   config = mkIf cfg.enable {
     systemd.services.hpos-admin-api = {
-      after = [ "network.target" ];
-      wantedBy = [ "multi-user.target" ];
+      after = [ "network.target" "nginx.service" ];
+      wantedBy = [ "multi-user.target" "nginx.service" ];
```
Comment (Br1ght0ne) [inline modules/services/hpos-admin-api.nix:22]: There's no reason `nginx.service` would "want" this to be started. `wantedBy` makes sense only as a guide to when the service should be started (`multi-user.target` effectively means "system is accessible to multiple users == startup finished").

```suggestion
      wantedBy = [ "multi-user.target" ];
```

File: modules/services/hpos-holochain-api.nix [hunk 2]
```diff
@@ -18,8 +18,8 @@ in
 
   config = mkIf cfg.enable {
     systemd.services.hpos-holochain-api = {
-      after = [ "network.target" ];
-      wantedBy = [ "multi-user.target" ];
+      after = [ "network.target" "nginx.service" ];
+      wantedBy = [ "multi-user.target" "nginx.service" ];
```
Comment (Br1ght0ne) [inline modules/services/hpos-holochain-api.nix:22]:
```suggestion
```

**Review by Br1ght0ne — CHANGES_REQUESTED** [review]
Checks out, except for service configuration nitpicks.

File: modules/services/hpos-holochain-api.nix (hunk 2 above)
Comment (alastairong) [reply modules/services/hpos-holochain-api.nix:22]: LOL I misread that as `required = `. Will address

**Comment by alastairong** [comment]
Ironically, it failed tests. This was because Envoy took 12 seconds to get ready.

I will change from a sleep to a more intelligent wait, since I don't want to add a 15 second forced sleep whenever Envoy restarts

**Comment by alastairong** [comment]
@Br1ght0ne Can i get a re-review please? 

Updated Envoy service to wait until it has started the server, for a max of 1 minute
Also updated hpos-holochain-api to wait for holochain

The envoy handling is probably overkill normally. It takes ages on hydra but not when running locally.

**Review by Br1ght0ne — APPROVED** [review]
LGTM, thanks @alastairong!

**Pull request merged**